spec/compiler/config_spec.cr

```diff
@@ -0,0 +1,14 @@
+require "../spec_helper"
+require "./spec_helper"
+
+describe Crystal::Config do
+  it ".default_target" do
+    Crystal::Config.default_target.should eq {{ `crystal --version`.lines[-1] }}.lstrip("Default target: ")
```

Comment:
```suggestion
    Crystal::Config.default_target.to_s.should eq {{ `crystal --version`.lines[-1] }}.lstrip("Default target: ")
```

Reply:
`Crystal::Config.default_target` is already a String.

Reply:
nope, it's `Crystal::Codegen::Target`.